.gitignore

```diff
@@ -182,6 +182,7 @@ todo
 *.asm
 
 # Distribution files
+/dist/
```

Comment:
`/dist` should be uploaded when a new version release, so please revert this change.